src/factory.ts

```diff
@@ -200,44 +197,37 @@ export class Factory {
     return this.options.stave.space * spacing;
   }
 
-  Stave(paramsP: { x?: number; y?: number; width?: number; options?: Partial<StaveOptions> } = {}): Stave {
-    if (!this.options.renderer) throw new RuntimeError('NoRenderer');
+  Stave(params?: { x?: number; y?: number; width?: number; options?: StaveOptions }): Stave {
```

Comment:
Can we use default parameters here (in the destructured `params`)? E.g.,

```typescript
Stave(params: {x = 0; y = 0; width?: number; options?: StaveOptions}): Stave { ... }
```

Reply:
Apparently no.  I tried it and got.
```
A type literal property cannot have an initializer.ts(1247)
```
I think TS treats that right side of the colon as an interface / type. But instead of us giving it an interface name, we used a literal.

If we do the same thing with an interface (e.g., StaveOptions) we get a similar error:
```
An interface property cannot have an initializer.ts(1246)
```

That's probably because TS erases all types from the JS output (after it passes all its error checks).